### DISCUSSION TITLE
Chang log file location

### DISCUSSION THREAD

**Discussion by heremansgeert** [SafeExamBrowser/seb-win-refactoring#1021] [Q&A]
Is it possible to configure SEB to store logs outside the AppData folder in the users profile?

I want to log to a directory on the C drive that's accessible by the Azure Monitor Agent.

**Comment by dbuechel** [comment]
That is unfortunately currently not possible and will most likely not be changed in the near future, as it would require substantial changes to the logging functionalities.
